data/huion-new-1060-plus.tablet

```diff
@@ -0,0 +1,23 @@
+# HUION
+# New 1060 Plus
+#
+
+[Device]
+Name=Huion New 1060 Plus
+ModelName=
+DeviceMatch=usb:256c:006e:HID 256c:006e Pen;usb:256c:006e:HID 256c:006e Pad
+Class=Bamboo
+Width=10
+Height=6
+IntegratedIn=
+Layout=huion-new-1060-plus.svg
+
+[Features]
+Stylus=true
```

Comment:
Is it okay not to specify any stylii? It comes with a simple stylus with two buttons and no eraser. gnome shows my the two buttons but it says "General Pen Eraser".

Reply:
IIRC the huions don't support serial numbers (check for `MSC_SERIAL` in the `libinput record` output of the device). So in that case - use the generic 0xffffd stylus please - see the existing huion file(s) for reference.